keycloak-plugins/radius-disconnect-plugin/src/main/java/com/github/vzakharchenko/radius/dm/api/RadiusService.java

```diff
@@ -2,7 +2,6 @@
 
 import com.github.vzakharchenko.radius.dm.models.RadiusInfoModel;
 import com.github.vzakharchenko.radius.dm.models.RadiusServiceModel;
-import org.jboss.resteasy.annotations.cache.NoCache;
```

Comment:
could it replace by import org.jboss.resteasy.reactive.NoCache by adding below dependency
```
        <!-- https://mvnrepository.com/artifact/io.quarkus/quarkus-rest -->
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-rest</artifactId>
            <version>3.12.0</version>
        </dependency>
```

Reply:
The other thing is good for me.

Reply:
Thanks, fixed